src/components/cards/AppInfoCard.tsx

```diff
@@ -828,7 +832,11 @@ const AppInfoCard: React.FC<ComponentProps> = ({
 
         <CardContent>
           {isProgressBarVisible && (
-            <ProgressBar progress={progress} label="Datacap used" />
+            <ProgressBar
+              progress={progress}
+              label="Datacap used"
```

Comment:
As requested in the ticket:
```suggestion
              label="Datacap used from most recent allocation"
```